specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,498 @@
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+New styling properties are being added to the OS Xaml CalendarView control in order to enable more flexible rendering. Because a large part of the CalendarView rendering is achieved through custom OS code, 
+the desired look cannot be achieved solely via XAML markup in the WinUI 2.6+ packages. New properties, used by those WinUI 2.6+ packages, are required to drive the custom OS rendering code.
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+A new set of CalendarView styling properties are introduced to allow greater customization of the control's rendering in upcoming Windows OS releases.
+These properties will be used in WinUI to align the CalendarView's styling with the rest of the standard controls.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+## Today styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the current day's rendering when CalendarView.IsTodayHighlighted is True:
+TodayBackground, TodayBlackoutBackground, TodayBlackoutForeground, TodayHoverBackground, TodayPressedBackground, TodaySelectedInnerBorderBrush.
+
+### Showcasing the TodayBackground property in the month, year and decade views
+![Default Today Background.](images/DefaultTodayBackground.png)
+![Default Today Month Background.](images/TodayMonthBackground.png)
+![Default Today Year Background.](images/TodayYearBackground.png)
+
+### Showcasing the TodayHoverBackground property - current date is hovered
+![Hover Today Background.](images/HoverTodayBackground.png)
+
+### Showcasing the TodayPressedBackground property - current date is pressed
+![Pressed Today Background.](images/PressedTodayBackground.png)
+
+### Showcasing the TodayBlackoutBackground and TodayBlackoutForeground properties - the old property CalendarView.IsTodayBlackedOut is True
```

Comment:
What does 'old' mean? As in deprecated, or meaning not new to this spec?

Reply:
We could use the term 'existing'. It's just not new in this spec.